src/server/main_service.cc

```diff
@@ -2161,6 +2161,18 @@ void Service::PubsubPatterns(ConnectionContext* cntx) {
   (*cntx)->SendLong(pattern_count);
 }
 
+void Service::PubsubNumSub(const vector<string_view>& channels, ConnectionContext* cntx) {
```

Comment:
Can you please add a test for this function in our connection test (under tests/dragonfly/connection_test.py`)?

Reply:
fixed: https://github.com/dragonflydb/dragonfly/pull/2282/commits/48cb68e5c9734eff009a150594d4d0d83716d2e7